datamap/mv/core/src/main/scala/org/apache/carbondata/mv/rewrite/SummaryDatasetCatalog.scala

```diff
@@ -39,6 +40,12 @@ private[mv] case class SummaryDataset(signature: Option[Signature],
     dataMapSchema: DataMapSchema,
     relation: ModularPlan)
 
+case class MVPlanWrapper(plan: ModularPlan, dataMapSchema: DataMapSchema) extends ModularPlan {
```

Comment:
Why is this needed? package visibility problem?

Reply:
No, it is to provide datamap schema information along with relation. Added comment as well.